lang/fr-fr/lang.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+return [
+    'plugin' => [
+        'name' => "CORS",
+        'description' => "Cross-Origin Resource Sharing.",
+    ],
+    'permissions' => [
+        'settings' => "Gestion de CORS",
+    ],
+    'settings' => [
+        'menu_label' => "CORS",
+        'menu_description' => "Configuration de CORS",
+        'fields' => [
+            'supportsCredentials' => [
+                'label' => "Support de credentials",
+                'comment' => "A activer pour supporter des credentials entre domaines."
+            ],
```

Comment:
Can you add the new field here?

`'paths' => [ 'label' => 'Paths', 'comment' => 'The url paths that will allow CORS.' ],`